Gothic-Unity-Project/Gothic-Unity
# Feature/106 alter mesh of backpack

> IMPORTANT: Test it now, but merge only after #141

## To test
* Boot the game normally (Production GameConfiguration is fine)
* Grab your backpack and see if the new mesh is visible
* Can you place items into? Then we're fine

![image](https://github.com/user-attachments/assets/5338174b-160b-454b-a7db-61b6e2571bb9)
